app/models/position_history.rb

```diff
@@ -0,0 +1,6 @@
+class PositionHistory < ApplicationRecord
+  validates :started_on, presence: true
+  validates :finished_on, presence: true
+  validates :employee_id, uniqueness: { scope: [:position_id] }
```

Comment:
No needs for validates `employee_id` and `position_id`.
It should be mentioned like: `belongs_to :employee` and `belongs_to :position`.
But first of all, let's merge the other PRs and then you will be able to do this.